file-formats/cota/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -0,0 +1,80 @@
+INTERFACE zif_aff_cota_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Connection Target Object</p>
+    "! Creation of connection target object
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version        TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Connection Target Object ID</p>
+      "! Connection target object ID
+      conn_id               TYPE c length 30,
```

Comment:
I guess this is the object name, isn't it? If so, please remove it. The name will be specified by the file name.